src/SamplesApp/SamplesApp.net6mobile/SamplesApp.net6mobile.csproj

```diff
@@ -34,7 +34,39 @@
 	<PropertyGroup>
 		<SynthesizeLinkMetadata>true</SynthesizeLinkMetadata>
 	</PropertyGroup>
-	
+
+	<Choose>
+
+		<When Condition="'$(TargetFramework)'=='net6.0-ios'">
+			<PropertyGroup Condition="'$(TargetFramework)'=='net6.0-ios'">
+				<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
+
+				<!-- See https://github.com/unoplatform/uno/issues/9430 for more details. -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --registrar:static</MtouchExtraArgs>
+
+				<!-- https://github.com/xamarin/xamarin-macios/issues/14812 -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --marshal-objectivec-exceptions:disable</MtouchExtraArgs>
+			</PropertyGroup>
+			<ItemGroup>
+				<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.4.0" />
+			</ItemGroup>
+		</When>
+
+		<When Condition="'$(TargetFramework)'=='net6.0-maccatalyst'">
```

Comment:
interesting, I forgot to check catalyst...

should not `build/uno.winui.common.targets` also have logic to ensure it's present ?
and that one should be for `maccatalyst-x64` and `maccatalyst-arm64`

Reply:
I originally did not add it because people are using the template which already has that enabled. It's mostly to catch existing projects that don't have the registrar setting.